### PR TITLE
Adding valuing people-process image

### PR DESCRIPTION
**Pull request by EKaroune**
### Summary

Adding people-process image to chapater in community handbook about community and shared ownership.

Fixes #

### List of changes proposed in this PR (pull-request)

- [x] adding image file to folder
- [x] adding code and alt text to page file

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/community-handbook/acknowledgement/acknowledgement-members.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ As a community-led project, we believe in and promote a collaborative approach t
 For example, writing _The Turing Way_ chapter is an iterative process that involves designing, outlining, writing, reviewing, and editing its contents.
 Similarly, community-oriented processes evolve over time with the help and feedback from the community members.
 
+```{figure} ../../figures/people-process.*
+---
+height: 500px
+name: people-process
+alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Underneath the ground are three roots - each with an image of a person at the end. The root images are a lady presenting a talk, a man in a lab coat collecting data and a person with glasses working at a laptop computer. They are representing the hidden work that goes into research.
```
Comment (EKaroune) [inline book/website/community-handbook/acknowledgement/acknowledgement-members.md:12]:
```suggestion
alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Underneath the ground are three roots - each with an image of a person at the end. The root images are a femaile-presenting person presenting a talk, a male-presenting person with a moustache in a lab coat collecting data and a person with glasses working at a laptop computer. They are representing the hidden work that goes into research.
```

File: book/website/community-handbook/acknowledgement/acknowledgement-members.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ As a community-led project, we believe in and promote a collaborative approach t
 For example, writing _The Turing Way_ chapter is an iterative process that involves designing, outlining, writing, reviewing, and editing its contents.
 Similarly, community-oriented processes evolve over time with the help and feedback from the community members.
 
+```{figure} ../../figures/people-process.*
+---
+height: 500px
+name: people-process
+alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Underneath the ground are three roots - each with an image of a person at the end. The root images are a femaile-presenting person presenting a talk, a male-presenting person with a moustache in a lab coat collecting data and a person with glasses working at a laptop computer. They are representing the hidden work that goes into research.
```
Comment (EKaroune) [inline book/website/community-handbook/acknowledgement/acknowledgement-members.md:12]:
```suggestion
alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Underneath the ground are three roots - each with an image of a person at the end. The root images are a female-presenting person presenting a talk, a male-presenting person with a moustache in a lab coat collecting data and a person with glasses working at a laptop computer. They are representing the hidden work that goes into research.
```

**Review by RichardJActon — COMMENTED** [review]
the people process image was from version 9 of the zenodo repo and the DOI link is to the generic DOI should it be general or version specific?

**Comment by EKaroune** [comment]
the general doi is ok

File: book/website/community-handbook/acknowledgement/acknowledgement-members.md [hunk 3]
```diff
@@ -5,6 +5,15 @@ As a community-led project, we believe in and promote a collaborative approach t
 For example, writing _The Turing Way_ chapter is an iterative process that involves designing, outlining, writing, reviewing, and editing its contents.
 Similarly, community-oriented processes evolve over time with the help and feedback from the community members.
 
+```{figure} ../../figures/people-process.*
+---
+height: 500px
+name: people-process
+alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Underneath the ground are three roots - each with an image of a person at the end. The root images are a female-presenting person presenting a talk, a male-presenting person with a moustache in a lab coat collecting data and a person with glasses working at a laptop computer. They are representing the hidden work that goes into research.
```
Comment (BatoolMM) [inline book/website/community-handbook/acknowledgement/acknowledgement-members.md:12]:
```suggestion
alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Below the ground, three roots extend, each terminating with an illustrated figure symbolising the often hidden efforts in research. From left to right: a female presenter giving a talk, a mustached male figure in a lab coat collecting data, and a person with glasses diligently working on a laptop. These figures represent the unseen work integral to the research process.
```

**Review by BatoolMM — APPROVED** [review]
I made very minor suggestion, feel free to ignore it. BTW, I love this illustration!

File: book/website/community-handbook/acknowledgement/acknowledgement-members.md [hunk 4]
```diff
@@ -5,6 +5,15 @@ As a community-led project, we believe in and promote a collaborative approach t
 For example, writing _The Turing Way_ chapter is an iterative process that involves designing, outlining, writing, reviewing, and editing its contents.
 Similarly, community-oriented processes evolve over time with the help and feedback from the community members.
 
+```{figure} ../../figures/people-process.*
+---
+height: 500px
+name: people-process
+alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Below the ground, three roots extend, each terminating with an illustrated figure symbolising the often hidden efforts in research. From left to right: a female presenter giving a talk, a mustached male figure in a lab coat collecting data, and a person with glasses diligently working on a laptop. These figures represent the unseen work integral to the research process.
```
Comment (BatoolMM) [inline book/website/community-handbook/acknowledgement/acknowledgement-members.md:12]:
```suggestion
alt: Cartoon-like sketch with pink colours of a large sunflower with a star in the middle sitting on top of the ground. It has the text next to the flower that says 'valuing people and processes behind research'. Below the ground, three roots extend, each terminating with an illustrated figure symbolising the often hidden efforts in research. From left to right, a female presenter giving a talk, a mustached male figure in a lab coat collecting data, and a person with glasses diligently working on a laptop. These figures represent the unseen work integral to the research process.
```

Comment (BatoolMM) [reply book/website/community-handbook/acknowledgement/acknowledgement-members.md:12]: Just removing the ":", hopefully it will do the trick!

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by EKaroune** [comment]
@BatoolMM please now approve as all looks to be working.

**Comment by EKaroune** [comment]
@all-contributors please add @RichardJActon for review

**Comment by EKaroune** [comment]
@all-contributors please add @kallewesterling for bug

**Review by BatoolMM — APPROVED** [review]
Approved @EKaroune - feel free to merge it

**Pull request merged**